ospfd/ospf_vty.c

```diff
@@ -7998,8 +7998,8 @@ DEFUN (ip_ospf_area,
 	argv_find(argv, argc, "area", &idx);
 	areaid = argv[idx + 1]->arg;
 
-	if (ifp->vrf_id && !instance)
-		ospf = ospf_lookup_by_vrf_id(ifp->vrf_id);
+	if (ifp->vrf && !instance)
```

Comment:
How could ifp->vrf be null?  This breaks the test of what was being done here.

Reply:
modified a bit